backend/backend/api/routes.py

```diff
@@ -130,12 +130,12 @@ async def read_root():
     </style>
```

Comment:
Consider renaming the input fields in the HTML form to better match their purpose. For example,